server/plugin/subscriptions.go

```diff
@@ -129,6 +138,15 @@ func (s *Subscription) RenderStyle() string {
 	return s.Flags.RenderStyle
 }
 
+func (s *Subscription) channelExcludedForRepo(repo *github.Repository) bool {
```

Comment:
I don't understand the use of this function name here. Where is the use of `channel`?